phoenix-core/src/it/java/org/apache/phoenix/end2end/SystemCatalogCreationOnConnectionIT.java

```diff
@@ -658,4 +670,30 @@ private Properties getClientProperties(boolean nsMappingEnabled, boolean systemT
         assertFalse(isSystemNamespaceCreated());
         return driver;
     }
+
+    @Test
+    public void testDoNotUpgradePropSet() throws Exception {
```

Comment:
nit: Add comments like in other test cases as a header for the expected behavior for ease of understanding.

Reply:
Added comments here.